app/code/Magento/Inventory/Model/Indexer/StockItem/DataProvider.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Model\Indexer\StockItem;
+
+use Magento\Framework\App\ResourceConnection;
+
+/**
+ * Stock Item Dimension
+ */
+class DataProvider
+{
+
+    const TABLE_NAME_SOURCE_ITEM = 'inventory_source_item';
+    const TABLE_NAME_STOCK_SOURCE_LINK = 'inventory_source_stock_link';
```

Comment:
Please describe constants.
More info is available [in the devdocs](http://devdocs.magento.com/guides/v2.2/coding-standards/docblock-standard-general.html#constants)